android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -301,6 +313,11 @@ public ReactExoplayerView(ThemedReactContext context, ReactExoplayerConfig confi
         themedReactContext.addLifecycleEventListener(this);
         audioBecomingNoisyReceiver = new AudioBecomingNoisyReceiver(themedReactContext);
         audioFocusChangeListener = new OnAudioFocusChangedListener(this, themedReactContext);
+        pictureInPictureReceiver = new PictureInPictureReceiver(this, themedReactContext);
```

Comment:
Is it necessary to allocate it every time ? in case of multiple video, I am afraid of mixing pip... 
BTW, It question me on do it really make sens to do it on react native video side ?

Reply:
I think play/pause control is common and essential. (If it were native, it would be easy to customize, but it's hard to make it controllable with a prop.)

Reply:
You didn't get my comment, If the app doesn't want to use pip, can we avoid allocating pictureInPictureReceiver ?

BTW, application may way to remove all buttons like play/pause when playing live content without backbuffer ! but I don't think this is mandatory to support this feature for the first PR